### PR TITLE
Run code formatter on CodeTest

### PR DESCRIPTION
**Pull request by rlustin**

### CODE REVIEW

File: lib/elixir/test/elixir/code_test.exs [hunk 1]
```diff
@@ -74,7 +75,11 @@ defmodule CodeTest do
 
   test "eval_quoted/2 with a %Macro.Env{} struct as the second argument" do
     alias :lists, as: MyList
-    assert Code.eval_quoted(quote(do: MyList.flatten [[1, 2, 3]]), [], __ENV__) == {[1, 2, 3], []}
+
+    assert Code.eval_quoted(quote(do: MyList.flatten([[1, 2, 3]])), [], __ENV__) == {
+             [1, 2, 3],
+             []
+           }
```
Comment (josevalim) [inline lib/elixir/test/elixir/code_test.exs]: Please format this one and the others below like this:

```elixir
assert Code.eval_quoted(quote(do: MyList.flatten([[1, 2, 3]])), [], __ENV__) ==
           {[1, 2, 3], []}
```

Then you run the formatter again and the formatter should keep the formatting you adjusted.

File: lib/elixir/test/elixir/code_test.exs [hunk 2]
```diff
@@ -101,16 +106,38 @@ defmodule CodeTest do
...
-           {:ok, {:sigil_r, [terminator: "/", line: 1], [{:<<>>, [line: 1], ["foo"]}, []]}}
-    assert Code.string_to_quoted("~r[foo]") ==
-           {:ok, {:sigil_r, [terminator: "[", line: 1], [{:<<>>, [line: 1], ["foo"]}, []]}}
-    assert Code.string_to_quoted("~r\"foo\"") ==
-           {:ok, {:sigil_r, [terminator: "\"", line: 1], [{:<<>>, [line: 1], ["foo"]}, []]}}
-    assert Code.string_to_quoted("~S\"\"\"\nsigil heredoc\n\"\"\"") ==
-           {:ok, {:sigil_S, [terminator: "\"\"\"", line: 1], [{:<<>>, [line: 1], ["sigil heredoc\n"]}, []]}}
-    assert Code.string_to_quoted("~S'''\nsigil heredoc\n'''") ==
-           {:ok, {:sigil_S, [terminator: "'''", line: 1], [{:<<>>, [line: 1], ["sigil heredoc\n"]}, []]}}
+    assert Code.string_to_quoted("~r/foo/") == {
```
Comment (josevalim) [inline lib/elixir/test/elixir/code_test.exs]: Same here and below.

File: lib/elixir/test/elixir/code_test.exs [hunk 3]
```diff
@@ -126,40 +153,48 @@ defmodule CodeTest do
...
+    assert string_to_quoted.("0xEF") == {:__block__, [original: '0xEF', line: 1], [239]}
+    assert string_to_quoted.("12.3") == {:__block__, [original: '12.3', line: 1], [12.3]}
+    assert string_to_quoted.("nil") == {:__block__, [line: 1], [nil]}
+    assert string_to_quoted.(":one") == {:__block__, [line: 1], [:one]}
+
+    assert string_to_quoted.("[1]") == {
+             :__block__,
+             [end_line: 1, line: 1],
+             [[{:__block__, [original: '1', line: 1], [1]}]]
+           }
```
Comment (josevalim) [inline lib/elixir/test/elixir/code_test.exs]: This one would need to break into the previous line. something like:

```elixir
args = [[{:__block__, [original: '1', line: 1], [1]}]]
assert string_to_quoted.("[1]") ==
       {:__block__, [end_line: 1, line: 1], block}
```

Do the same below and please format the file again.

File: lib/elixir/test/elixir/code_test.exs (hunk 1 above)
Comment (rlustin) [reply lib/elixir/test/elixir/code_test.exs]: The code formatter keeps format the code as in the PR. I can’t do what you ask, or I misunderstood something :)

File: lib/elixir/test/elixir/code_test.exs (hunk 2 above)
Comment (rlustin) [reply lib/elixir/test/elixir/code_test.exs]: Same here and below.

File: lib/elixir/test/elixir/code_test.exs (hunk 3 above)
Comment (rlustin) [reply lib/elixir/test/elixir/code_test.exs]: Fixed.

File: lib/elixir/test/elixir/code_test.exs (hunk 1 above)
Comment (josevalim) [reply lib/elixir/test/elixir/code_test.exs]: It means your formatter is out of date. Please rebase/merge latest master, run `make clean compile` and then it should keep the formatting proposed.

Comment (rlustin) [reply lib/elixir/test/elixir/code_test.exs]: That’s what I’ve missed. `clean` to recompile the formatter. Thanks!

Comment (rlustin) [reply lib/elixir/test/elixir/code_test.exs]: Fixed now.

File: lib/elixir/test/elixir/code_test.exs (hunk 2 above)
Comment (rlustin) [reply lib/elixir/test/elixir/code_test.exs]: Fixed now.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**